sessions-core/src/main/java/com/adyen/checkout/sessions/CheckoutSession.kt

```diff
@@ -8,9 +8,16 @@
 
 package com.adyen.checkout.sessions
 
+import com.adyen.checkout.components.model.paymentmethods.PaymentMethod
 import com.adyen.checkout.sessions.model.setup.SessionSetupResponse
 
 // TODO docs
 data class CheckoutSession(
     val sessionSetupResponse: SessionSetupResponse
-)
+) {
+    fun getPaymentMethod(paymentMethodType: String): PaymentMethod? {
```

Comment:
Shall we add this helper for stored payment methods too?
```kotlin
    fun getStoredPaymentMethod(predicate: (StoredPaymentMethod) -> Boolean): StoredPaymentMethod? {
        return sessionSetupResponse.paymentMethods?.storedPaymentMethods.orEmpty().firstOrNull {
            predicate(it)
        }
    }
```

Reply:
Why do we need the predicate again?

Reply:
For stored payment methods we don't know what they're gonna use to choose the payment method they want. I think this is the best option we can provide for that so they can pass the condition they desire.